README.md

```diff
@@ -1,194 +1,122 @@
 
-```typescript
-const profile = await client.getProfile('<profileName>');
+```shell
+npx @superfaceai/cli install <profileName>
 ```
 
-Next, make sure at least one provider is configured in super.json or select one manually. You can configure providers in super.json by running `superface configure <providerName>` and you can add additional or overriding configuration by calling `.configure` on the Provider object:
+Next you configure a provider for the use-case, optionally with API keys:
 
-```typescript
-const provider = await client.getProvider('<providerName>');
-// provider.configure(...)
+```shell
+npx @superfaceai/cli configure <providerName> -p <profileName>
 ```
 
-Then, obtain a use case and perform it with selected provider:
-
-```typescript
-const result = await profile.getUsecase('<usecaseName>').perform(
-  {
-    inputField: 1,
-    anotherInputField: 'hello',
-  },
-  // optional, if provider is missing selects first configured provider from super.json
-  { provider }
-);
-```
-
-### Handling the results from `perform`
+The CLI creates a configuration file in `superface/super.json` file which is loaded by OneSDK.
 
-The `perform` method will take your inputs and additional information and perform the use case asynchronously. This method always returns a Result type that is either `Ok` or `Err`. This follows the [neverthrow](https://github.com/supermacro/neverthrow) approach. The SDK provides multiple ways to work with result. 
+In your code, you initialize the SDK instance, load the profile and perform the use-case:
 
-#### Conditionals
+```js
+const { SuperfaceClient } = require('@superfaceai/one-sdk');
 
-You can use conditionals to check if the result was OK or if it errored. Use `isOk()` or `isErr()`to check type of result.
+const sdk = new SuperfaceClient();
 
-```typescript
-if (result.isErr()) {
-  // Result is error, error.toString() returns human readable description of what went wrong
-  console.log(result.error.toString());
-} else {
-  // Result is ok and you can accees value here
-  console.log(result.value);
-}
-```
+async function run() {
+  const profile = await sdk.getProfile('<profileName>');
 
-#### Matching a value or error
+  const result = await profile.getUseCase('<usecaseName').perform({
+    // Input parameters
+  });
 
-The Result type also provides a `match` method to use functions to use the values or errors. The `match` method takes two functions, the first of which is for handling the `Ok` result and the the second for handling the `Err` result. The example above using `isOk` and `isErr` can be written using `match` like below.
+  console.log(result.unwrap());
+}
 
-```typescript
-result.match(
-  value => console.log(value),
-  error => console.log(error.toString())
-);
+run();
 ```
 
-#### Unsafely unwrapping the result
+Optionally you can [define which provider should be used](https://superface.ai/docs/guides/using-multiple-providers).
```

Comment:
Change it to "This code will use the provider specified in the super.json. You can override the configure in the code like this and that" 


----

bottom line: I feel you are referring too much to external docs. Ideally, at least the basic useage should be completely covered by the README.